charts/ndb-operator/README.md

```diff
@@ -0,0 +1,171 @@
+| `replicaCount`              | Number of replicas of the NDB Operator controller pods           | `1`                                                              |
+| `image.repository`          | Image for NDB Operator controller                                | `ghcr.io/nutanix-cloud-native/ndb-operator/controller`           |
+| `image.pullPolicy`          | Image pullPolicy                                                 | `IfNotPresent`                                                   |
+| `image.tag`                 | Image tag                                                        | `v0.0.2, defaults to Chart.appVersion if removed`                |
+| `imagePullSecrets`          | ImagePullSecrets list                                            | `[]`                                                             |
+| `nameOverride`              | To override the name of the operator chart                       | `""`                                                             |
+| `fullnameOverride`          | To override the full name of the operator chart                  | `""`                                                             |
+| `serviceAccount.name`       | Name of the service account that will be used by the operator    | `ndb-operator-service-account`                                   |
+| `podAnnotations`            | Add annotation to NDB Operator controller pods                   | `kubectl.kubernetes.io/default-container: manager`               |
+| `podSecurityContext`        | Security context for the pod(s) running the operator             | `runAsNonRoot: true`                                             |
+| `securityContext`           | Security context for the container running the controller        | `allowPrivilegeEscalation: false`                                |
+| `resources`                 | Configure resources for Cloud Provider Pod                       | `refer to values.yaml`                                           |
+| `nodeSelector`              | Configure nodeSelector for Cloud Provider Pod                    | `refer to values.yaml`                                           |
+| `tolerations`               | Configure tolerations for Cloud Provider Pod                     | `refer to values.yaml`                                           |
+| `affinity`                  | Configure affinity for Cloud Provider Pod                        | `refer to values.yaml`                                           |
+
+Specify each parameter using the `--set key=value[,key=value]` argument to `helm install` or provide a file with `-f value.yaml`.
+
+### Configuration examples:
+
+Install the operator in the `ndb-operator-ns` namespace (add the `--create-namespace` flag if the namespace does not exist): 
+
+```console
+helm install ndb-operator nutanix/ndb-operator -n ndb-operator-ns 
+```
+
+Individual configurations can be set by using `--set key=value[,key=value]` like:
```

Comment:
duplicate of line 117 ?

Reply:
Line 117 just explains that individual parameters can be set using the command, line 127-130 shows an example for the same.

Reply:
ok you choice :-)